Sources/Core/Internal/Scene+MultipleAdd.swift

```diff
@@ -0,0 +1,22 @@
+/**
+ *  Imagine Engine
+ *  Copyright (c) John Sundell 2017
+ *  See LICENSE file for license
+ */
+
+import Foundation
+
+extension Scene {
```

Comment:
I'm not sure this warrants its own file 🤔 Personally, I prefer to keep things in the same place as much as possible, since it makes navigating the code base a bit simpler. There's already an extension at the bottom of `Scene.swift` for convenience APIs (it currently only contains `center`), I suggest putting these there instead 🙂

Reply:
OK sounds good! I'll get to it later today

Reply:
Perfect, thanks 👍